tests/test_track_changes_coverage.py

```
@@ -0,0 +1,373 @@
+        xml_path.write_text(xml)
+        return xml_path
+
+    return _create_xml
+
+
+def _make_manager(xml_path: Path) -> RevisionManager:
+    editor = DocxXMLEditor(xml_path, rsid="00000000", author="Test Author")
+    return RevisionManager(editor)
+
+
+def _get_text_content(manager):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_get_text_content`

(ANN202)